raiden/raiden_event_handler.py

```diff
@@ -122,10 +123,10 @@ def unlock_light(raiden: "RaidenService",
         sender=partner,
         merkle_tree_leaves=leaves_packed
     )
-    if not LightClientMessageHandler.is_message_already_stored(
+    if not LightClientMessageHandler.get_message_by_content(
         light_client_address=channel_unlock_event.client,
         message_type=LightClientProtocolMessageType.UnlockLightRequest,
-        unsigned_message=message,
+        message=message,
```

Comment:
the thing is that the parameter has the message actually, the content is extracted inside the method

Reply:
i don't actually see this context being extracted, just the entire message being serialized. 

could you please expand on this?

Reply:
the serialized message is not the same as the object, the serialized message is a string, if you use this method with a string instead of a message object it will fail, so that's why i didn't use the message_content as a parameter because it could be confusing for who is going to use this

Reply:
it's not a blocker for me. the issue i have here is that if the method is called `get_message_by_content`, it should be clear what the content is. the type is one thing, and the serialized message is another, so what is actually the _content_?

Reply:
The content should be the serialized message value, the thing is you can't provide that from outside the method, only from the sqlite instance, since we have the serializer there.